From: Michael Tokarev <mjt@tls.msk.ru>
Subject: re-enable some ps config options for DESKTOP case
Forwarded: no, should forward

commit 8d9ac30572818bbe78ef08d6580308e013972df3
Author: Denys Vlasenko <vda.linux@googlemail.com>
Date:   Mon Sep 26 02:56:08 2011 +0200

    ps: add support for -l for !DESKTOP
    
made FEATURE_PS_WIDE option depending on !DESKTOP, thus
effectively disabling it for desktop case.  Re-enable it
and the new FEATURE_PS_LONG too.

--- a/procps/Config.src
+++ b/procps/Config.src
@@ -90,7 +90,7 @@ config PS
 config FEATURE_PS_WIDE
 	bool "Enable wide output option (-w)"
 	default y
-	depends on PS && !DESKTOP
+	depends on PS
 	help
 	  Support argument 'w' for wide output.
 	  If given once, 132 chars are printed, and if given more
@@ -99,7 +99,7 @@ config FEATURE_PS_WIDE
 config FEATURE_PS_LONG
 	bool "Enable long output option (-l)"
 	default y
-	depends on PS && !DESKTOP
+	depends on PS
 	help
 	  Support argument 'l' for long output.
 	  Adds fields PPID, RSS, START, TIME & TTY
